Dockerfile

```diff
@@ -12,16 +12,16 @@ ENV DEBIAN_FRONTEND noninteractive
 
 
 # Adapted from: https://registry.hub.docker.com/u/jpetazzo/dind/dockerfile/
-RUN apt-get update -qq
-RUN apt-get install -qqy apt-transport-https
-
-# Install Docker from Docker Inc. repositories
-RUN echo deb https://get.docker.io/ubuntu docker main > /etc/apt/sources.list.d/docker.list
-RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 36A1D7869245C8950F966E92D8576A8BA88D21E9
-RUN apt-get update -qq && \
-    apt-get install -y apparmor ca-certificates iptables lxc-docker && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+# Let's start with some basic stuff.
+RUN apt-get update -qq && apt-get install -qqy \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    lxc \
+    iptables
```

Comment:
* Do we not need "apparmor"? I seem to recall needing it. Can you confirm that this image works as a Docker/Jenkins slave without apparmor?
* Was the removal of the `rm -rf /var/lib/apt/lists/*` intentional?